src/Testcontainers.Ollama/OllamaContainer.cs

```diff
@@ -0,0 +1,52 @@
+namespace Testcontainers.Ollama
+{
+    /// <inheritdoc cref="DockerContainer" />
+    [PublicAPI]
+    public sealed class OllamaContainer : DockerContainer
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OllamaContainer" /> class.
+        /// </summary>
+        /// <param name="configuration">The container configuration.</param>
+        /// <param name="logger">The logger.</param>
+        public OllamaContainer(OllamaConfiguration configuration, ILogger logger)
+            : base(configuration, logger)
+        {
+            ModelName = configuration.ModelName;
+            ImageName = OllamaConfiguration.ImageName;
+        }
+    
+        /// <summary>
+        /// Starts the Ollama container.
+        /// </summary>
+        public async Task StartOllamaAsync()
+        {
+            if (State!= TestcontainersStates.Created && State != TestcontainersStates.Running)                {
+                throw new InvalidOperationException("Cannot start a container that has not been created.");
+            }
+            Task.WaitAll(ExecAsync(new List<string>()
+            {
+                "ollama", "run", ModelName,
+            }));
+            
+            await Task.CompletedTask;
+        }
```

Comment:
Does it make sense to pass the model name as an argument to the member? `Run(string modelName, CancellationToken ct = default)`? In addition, please do not block the thread; simply return `ExecAsync(["ollama", "run", modelName], ct);`.

Reply:
Agreed and resolved about the return. I added two methods one taking only the ct and one taking name and ct. Have a look if you agree